test_unstructured/metrics/test_text_extraction.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from unstructured.metrics.text_extraction import calculate_edit_distance
+from unstructured.partition.auto import partition
+
+
+@pytest.mark.parametrize(
+    ("filename", "expected_score", "expected_distance"),
+    [
+        ("fake-text.txt", 0.82, 30),
+        ("eml/fake-email.eml", 0.03, 164),
+        ("README.md", 0.0, 719),
+    ],
+)
+def test_calculate_edit_distance(filename, expected_score, expected_distance):
+    with open("example-docs/fake-text.txt") as f:
```

Comment:
are all of the parameterized files being compared to the same fake-text.txt source cct? that explains why the scores are so low! let's compare each file to it's own source cct (can be made by hand if its just a few documents)

Reply:
I believe we do not have the CCT function ready yet so I made tests possible as I could. (We have ground truth/gold standard, but not the function to convert Unstructured elements to CCT if I understand correctly).

Reply:
Ok, as mentioned previously we can test this function on some smaller string cases as well as files where it's easy to create a human transcription of the text. We definitely don't want to be mismatching source and output files

Reply:
Let's either remove "eml/fake-email.eml" and "README.md" from the list of filenames being tested, or create a source cct for them to be compared to (I'd pick the first option, second is too much work for now). We should not be comparing CCT to a different source file.